ha/cli/displayOutput.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import json
+#import sys
+
+
+class Output():
+    """Class representing a generic framework for output handling. """
+
+    #def __init__(self, rc, desc):
+        #self.output_string = "output"
```

Comment:
Why is this commented?

Reply:
This portion of code will be addressed in the https://jts.seagate.com/browse/EOS-18599 ticket